src/ir/LocalGraph.cpp

```diff
@@ -61,6 +61,14 @@ struct Info {
   }
 };
 
+// Minimalist block information needed
+struct OptimizedBlock {
+  size_t traverseIdx; // last Traversed Index
```

Comment:
This seems to count the number of iterations in the algorithm, but I'm not sure?

The term `Index` in this codebase implies the index of a local, so perhaps another term is better here. `iteration`?

Reply:
In the main loop `foreach numLocals` I increment a reference counter.
Then while I traverse the optimized block struct I compare the `traverseIdx` to ensure I didn't see this block in this loop.

So, yes it `size_t traverseIteration; \\ last Traversed Iteration` should be better.